Source/Core/DolphinWX/ISOProperties.cpp

```diff
@@ -504,6 +514,26 @@ void CISOProperties::CreateGUIControls(bool IsWad)
 	sConfigPage->Add(sEmuState, 0, wxEXPAND|wxALL, 5);
 	m_GameConfig->SetSizer(sConfigPage);
 
+	// Remove Object
+	wxBoxSizer* const sRmObjs = new wxBoxSizer(wxVERTICAL);
+
+	RmObjs = new wxCheckListBox(m_RmObjPage, ID_RMOBJS_LIST, wxDefaultPosition, wxDefaultSize, arrayStringFor_RmObjs, wxLB_HSCROLL);
+	wxBoxSizer* const sRmObjsButtons = new wxBoxSizer(wxHORIZONTAL);
+	EditRmObj = new wxButton(m_RmObjPage, ID_EDITRMOBJ, _("Edit..."));
+	wxButton* const AddRmObj = new wxButton(m_RmObjPage, ID_ADDRMOBJ, _("Add..."));
+	RemoveRmObj = new wxButton(m_RmObjPage, ID_REMOVERMOBJ, _("Remove"));
+	EditRmObj->Enable(false);
+	RemoveRmObj->Enable(false);
```

Comment:
For this I was maintaining consistency with CISOProperties::PatchButtonClicked and CISOProperties::ActionReplayButtonClicked, which both use the same Edit*->Enable(false) and Remove*->Enable(false).  Should I also update them all at the same time, or would it better to do it later in it's own pull?